using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md

```diff
@@ -170,8 +190,14 @@ ACPCampaign.resetLinkageFields();
 
 ## Set up push messaging
 
+To enable push messaging with Adobe Campaign, the push identifier received from APNS (Apple Push Notification Service) or FCM (Firebase Cloud Messaging) must be sent to the Adobe Identity service by invoking `ACPCore::setPushIdentifer`. This API should also be invoked if there is any change in the push identifier (for example when a new token is generated due to the previous token being expired). After the push identifier is sent to the Identity service, Campaign will then retrieve the push identifier from the Identity service directly.
```

Comment:
Please reword the 1st sentence as follows:
"To enable push messaging with Adobe Campaign, the push identifier from Apple Push Notification Service  (APNS ) or Firebase Cloud Messaging (FCM) must invoke `ACPCore::setPushIdentifer`, so it can be sent to the Adobe Identity service. "